src/matches.rs

```diff
@@ -246,8 +246,18 @@ fn rewrite_match_arm(
     };
 
     // Patterns
-    // 5 = ` => {`
-    let pat_shape = shape.sub_width(5)?.offset_left(pipe_offset)?;
+    let pat_shape = match &arm.body.kind {
+        ast::ExprKind::Block(_, Some(label)) => {
+            // Some block with a label ` => 'label: {`
+            // 7 = ` => : {`
+            let label_len = label.ident.as_str().len();
```

Comment:
Does the identifier string include the `'` prefix? If not we should account for that token in the width ourselves

Reply:
I double checked when I was writing the code and, yes the label contains the `'`.

Reply:
Maybe we should future proof the code by adding a check for the leading `'` in case that changes in the future